next_generation/tests/unit/mock_pint_data.py

```diff
@@ -0,0 +1,89 @@
+import glob
+import os
+
+from lxml import etree
+
+def images(provider):
+    base_path = os.path.dirname(os.path.abspath(__file__))
+    data_file = glob.glob(base_path + '/./mockpintdata/' + provider + '.xml')
```

Comment:
Isnt `/./mockpintdata` the same as `/mockpintdata` ? or am I missing something

Reply:
Updated